Compile on Windows error: Reference Xamarin.Mac not resolved

I'm trying to build Monodevelop on [Appveyor](appveyor.com) with Visual Studio 2019 environment, but following error has occurred: 

```
[00:05:03]        "C:\projects\monodevelop\monodevelop\main\Main.sln" (default target) (1:2) ->
[00:05:03]        "C:\projects\monodevelop\monodevelop\main\src\core\MonoDevelop.Ide\MonoDevelop.Ide.csproj" (default target) (13:59) ->
[00:05:03]        (CoreCompile target) -> 
[00:05:03]          MonoDevelop.Ide.Fonts\FontChooserPanelWidget.cs(32,7): error CS0246: The type or namespace name 'AppKit' could not be found (are you missing a using directive or an assembly reference?) [C:\projects\monodevelop\monodevelop\main\src\core\MonoDevelop.Ide\MonoDevelop.Ide.csproj]
[00:05:03]          MonoDevelop.Ide.Fonts\FontChooserPanelWidget.cs(33,7): error CS0246: The type or namespace name 'Foundation' could not be found (are you missing a using directive or an assembly reference?) [C:\projects\monodevelop\monodevelop\main\src\core\MonoDevelop.Ide\MonoDevelop.Ide.csproj]
[00:05:03] 
[00:05:03] 
[00:05:03]        "C:\projects\monodevelop\monodevelop\main\Main.sln" (default target) (1:2) ->
[00:05:03]        "C:\projects\monodevelop\monodevelop\main\tests\Ide.Tests\MonoDevelop.Ide.Tests.csproj" (default target) (63:8) ->
[00:05:03]        "C:\projects\monodevelop\monodevelop\main\tests\MonoDevelop.Core.Tests\MonoDevelop.Core.Tests.csproj" (default target) (80:8) ->
[00:05:03]          MonoDevelop.Projects\ProjectTests.cs(1016,54): error CS0234: The type or namespace name 'Composition' does not exist in the namespace 'System.ComponentModel' (are you missing an assembly reference?) [C:\projects\monodevelop\monodevelop\main\tests\MonoDevelop.Core.Tests\MonoDevelop.Core.Tests.csproj]
```
It seems like the Xamarin.Mac reference cannot be resolved.
The full log is attached below and the Appveyor repository is [here](https://ci.appveyor.com/project/wangkaibule/monodevelop)
[log (1).txt](https://github.com/mono/monodevelop/files/3774342/log.1.txt)

--- a/main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Fonts/FontChooserPanelWidget.cs
+++ b/main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Fonts/FontChooserPanelWidget.cs
@@ -29,10 +29,13 @@ using MonoDevelop.Core;
 using MonoDevelop.Components.AtkCocoaHelper;
 using Gtk;
 using System.Diagnostics;
-using AppKit;
-using Foundation;
 using MonoDevelop.Components;
 using Xwt.Drawing;
+
+#if MAC
+using AppKit;
+using Foundation;
+#endif
 
 namespace MonoDevelop.Ide.Fonts
 {

--- a/main/tests/MonoDevelop.Core.Tests/MonoDevelop.Core.Tests.csproj
+++ b/main/tests/MonoDevelop.Core.Tests/MonoDevelop.Core.Tests.csproj
@@ -20,6 +20,7 @@
     <Reference Include="Microsoft.CSharp" />
     <Reference Include="mscorlib" />
     <Reference Include="System.Net.Http" />
+    <Reference Include="System.ComponentModel.Composition" />
     <Reference Include="System.Configuration" />
     <!-- when building with xbuild on Unix, $(MSBuildToolsPath) is like `$prefix/lib/mono/xbuild/*/bin`
     when building with msbuild on Unix, $(MSBuildToolsPath) is like `$prefix/lib/mono/msbuild/*/bin`

